v4-client-py/v4_client_py/clients/dydx_composite_client.py

```diff
@@ -42,13 +43,25 @@ def __init__(
     def calculate_good_til_block(self) -> int:
         response = self.validator_client.get.latest_block()
         return response.block.header.height + 3
-    
+
     def calculate_good_til_block_time(self, good_til_time_in_seconds: int) -> int:
         now = datetime.now()
         interval = timedelta(seconds=good_til_time_in_seconds)
         future = now + interval
         return int(future.timestamp())
 
+    def validate_good_til_block(self, good_til_block: int) -> None:
```

Comment:
Ideally this code is updated such that it doesn't need to fetch the current block on each order placement. I just hacked this together to get it working for now.